ch6.md

```diff
@@ -300,31 +300,32 @@ foo( arr );
 console.log( arr[0] );			// 1
 ```
 
-Now `arr[0]` is quite reliably `1`.
+可以非常确定 arr[0] 就是 1.
 
-This is so important because it makes reasoning about our code much easier when we know we can trust that a value doesn't change when passed somewhere that we do not see or control.
+现在即使将值传递给一个并不知道其内部如何运作也无法控制其运作的的系统，我们也能够很简单的确定值是什么，这一点相当重要。
```

Comment:
1、相当重要的主语没有翻译正确，because后面的是非常重要的原因，而不是非常重要的主语
2、“现在即使将值传递给一个并不知道其内部如何运作也无法控制其运作的的系统，我们也能够很简单的确定值是什么”原文貌似没有这么多的意思， ->这是非常重要的因为这可以使理解我们的代码容易一些，当我们将值传递到我们看不到或者不能控制的地方，我们依然相信这个值不会改变